src/jni/jvm_loader.cpp

```diff
@@ -101,7 +102,22 @@ String jni::JvmLoader::get_embedded_jre_path() {
         };
         jre_path = vformat("%s%s", user_code_dir, LIB_GRAAL_VM_RELATIVE_PATH);
     } else {
-        jre_path = vformat("res://jre/%s", LIB_JVM_RELATIVE_PATH);
+        String jre_folder{
+                vformat(
+                        "%s%s",
+#if defined(OSX_ENABLED) && !defined(TOOLS_ENABLED)
+                        "../PlugIns/"
+#else
+                        ""
+#endif
+                        ,
+                        jni::JniConstants::CURRENT_RUNTIME_JRE
+                )
+        };
+        jre_path = OS::get_singleton()->get_executable_path()
```

Comment:
IMO we should use the `res://` folder here for everything that is not an export (like we had before)
Because when i run the game directly with the editor executable, it does not pick up the jre folder as the directory of the "executable" is where the editor executable lies.
Example:
![2022-08-12-151944_1332x174_scrot](https://user-images.githubusercontent.com/22662033/184362134-efc76f14-31e4-488b-9406-c81d57f9af8e.png)

Reply:
Fixed this by modifying a bit `jvm_loader.cpp`, can you re-test on linux ?